sgv2-restapi/pom.xml

```diff
@@ -259,6 +259,20 @@
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
       </plugin>
+      <!-- Plugin for building Docker images -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
```

Comment:
So what's the deal with the plugin and the `build.sh`/`Dockerfile` files, which one will be used?

Reply:
Note this is a draft PR and not intended for review yet. However, the plugin is the one that will be used, the Dockerfile and associated script will be deleted in the next push.